src/main/webapp/view/discussion/discussion.js

```diff
@@ -128,11 +58,11 @@ class DiscussionComment extends HTMLElement {
    *
    * @param comment The comment from the servlet that this element should
    *     render.
-   * @param {DiscussionManager} manager The current discussion's manager.
+   * @param {DiscussionArea} discussion The current discussion.
```

Comment:
nit: I feel like `The current discussion` is a bit vague.

Reply:
Changed